libs/db-browser/src/main/resources/schema/sql/oboracle/oboracle_4_1_x.yaml

```diff
@@ -0,0 +1,277 @@
+sqls:
+  list-basic-table-columns: |-
```

Comment:
we may figure out a way to avoid these too much duplicated sql statement in different yaml

Reply:
just suggestion, I may not be solved by this pr~

Reply:
i agree